include/deal.II/grid/grid_generator.h

```diff
@@ -176,6 +186,8 @@ namespace GridGenerator
    * z=0, defined by the two opposing corners @p p1 and @p p2.
    *
    * @note The triangulation passed as argument needs to be empty when calling this function.
+
+ 
```

Comment:
We should remove these two blank lines. This will also fix the complaint with the indentation engine.